slsDetectorSoftware/src/Module.h

```diff
@@ -123,6 +123,8 @@ class Module : public virtual slsDetectorDefs {
     void setFlipRows(bool value);
     bool isMaster() const;
     void setMaster(const bool master);
+    bool getSynchronization() const;
+    void setSynchronization(const bool value);
```

Comment:
const bool, is not having any effect in declaration only needed later in definition.

Reply:
but didnt we decide we will do this in a PR later where we remove all the const in declarations (except the ones with references)?

Reply:
Makes sense!